src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -224,7 +226,15 @@ class ThumbnailsPanel extends React.PureComponent {
   };
 
   onDocumentLoaded = () => {
-    const { setSelectedPageThumbnails } = this.props;
+    const { setSelectedPageThumbnails, dispatch } = this.props;
+
+    let doc = core.getDocument();
+    if (doc.type !== workerTypes.PDF) {
+      dispatch(actions.disableElements(['thumbnailControl',], PRIORITY_ONE));
```

Comment:
Is the `,` inside the array intentional?

Reply:
Thanks for pointing this out. I actually remove the "disableElements" here